docs/docker.md

```diff
@@ -0,0 +1,71 @@
+## Horovod in Docker
+
+To ease the installation process on GPU machines, we have published the reference [Dockerfile](../Dockerfile) that
+allows you to get started with Horovod in minutes. The container will include [Examples](../examples) in the `/examples`
+directory for a convenient quickstart.
+
+### Building
+
+```bash
+$ mkdir horovod-docker
+$ wget -O horovod-docker/Dockerfile https://raw.githubusercontent.com/uber/horovod/master/Dockerfile
+$ docker build -t horovod:latest horovod-docker
+```
+
+Before building, you can modify `Dockerfile` to your liking, e.g. select different CUDA, TensorFlow or Python version.
+
+### Running on a single machine
+
+After the container is built, you can run it using [nvidia-docker](https://github.com/NVIDIA/nvidia-docker).
+
+```bash
+$ nvidia-docker run -it horovod:latest
+root@c278c88dd552:/examples# mpirun -np 4 -H localhost:4 python keras_mnist_advances.py
+```
+
+You may notice that this command does not have few options recommended in other parts of documentation: 
+`-bind-to none -map-by slot -x NCCL_DEBUG=INFO -x LD_LIBRARY_PATH`.  These options are already set by default in the
+docker container, and so are not required to repeated in the command.
```

Comment:
docker --> Docker
", and so are not required to repeated in the command." --> "so you don't need to repeat them in the command."